CMakeSettings.json

```diff
@@ -2,97 +2,97 @@
   // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
   "configurations": [
     {
-      "name": "x64-Release",
-      "generator": "Visual Studio 15 2017 Win64",
+      "name": "x86-RelWithDebInfo",
+      "generator": "Visual Studio 15 2017",
       "configurationType": "RelWithDebInfo",
-      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "inheritEnvironments": [ "msvc_x64_x86" ],
       "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
       "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
       "cmakeCommandArgs": "",
-      "buildCommandArgs": "-m -v:minimal",
+      "buildCommandArgs": "",
       "variables": [
         {
-          "name": "OPENSSL_OFF",
+          "name": "INSTALLER_MERGE_MODULES",
```

Comment:
I suggest removing entries that are already cmake defaults so that cmake default changes implicitly apply here as well. I think only `CMAKE_BUILD_TYPE_INIT`, `WIN32`, `USE_SHARED_LIBS`, `DISABLE_CONTROLLER`, `FORCE_WINDOWS`, `DISABLE_SCRIPTING` and `EXCLUDE_BOOST` should stay.

Reply:
I think there is value in `CMakeSettings.json` being in sync with `win_build_vs.bat`; if we change a default, that will not automatically get applied there, either.  So I'd rather leave it like this.